lib/client/states/state.py

```diff
@@ -11,7 +11,7 @@ def leave(self):
         #buh bye - cleanup stuff
         pass
 
-    def render(self):
+    def render(self, screen):
```

Comment:
It's technically not now since we're not expecting them to be able to render onto subsurfaces. But is common UI lib practice especially once you have windows/panels.